spec/amp-email-actions-and-events.md

```diff
@@ -0,0 +1,500 @@
+    <a href="https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input#autofocus"><code>autofocus</code> element</a> becomes visible as a
+    result, it gains focus.</td>
+  </tr>
+  <tr>
+    <td><code>toggleClass(class=STRING, force=BOOLEAN)</code></td>
+    <td>Toggles class of the target element. <code>force</code> is optional, and if defined, it ensures that class would only be added but not removed if set to <code>true</code>, and only removed but not added if set to <code>false</code>.</td>
+  </tr>
+  <tr>
+    <td><code>focus</code></td>
+    <td>Makes the target element gain focus. To lose focus, <code>focus</code>
+    on another element (usually parent element). We strongly advise against
+    losing focus by focusing on <code>body</code>/<code>documentElement</code>
+    for accessibility reasons.</td>
+  </tr>
+</table>
+
+### amp-carousel[type="slides"] <a name="amp-carouseltypeslides-1"></a>
+
+<table>
+  <tr>
+    <th>Action</th>
+    <th>Description</th>
+  </tr>
+  <tr>
+    <td><code>goToSlide(index=INTEGER)</code></td>
+    <td>Advances the carousel to a specified slide index.</td>
+  </tr>
+</table>
+
+### amp-image-lightbox <a name="amp-image-lightbox"></a>
+
+<table>
+  <tr>
+    <th width="40%">Action</th>
+    <th>Description</th>
+  </tr>
+  <tr>
+    <td><code>open (default)</code></td>
+    <td>Opens the image lightbox with the source image being the one that triggered the action.</td>
+  </tr>
+</table>
+
+### amp-lightbox <a name="amp-lightbox-1"></a>
+
+<table>
+  <tr>
+    <th>Action</th>
+    <th>Description</th>
+  </tr>
+  <tr>
+    <td><code>open (default)</code></td>
+    <td>Opens the lightbox.</td>
+  </tr>
+  <tr>
+    <td><code>close</code></td>
+    <td>Closes the lightbox.</td>
+  </tr>
+</table>
+
+### amp-list <a name="amp-list-1"></a>
+
+<table>
+  <tr>
+    <th>Action</th>
+    <th>Description</th>
+  </tr>
+    <tr>
+    <td><code>changeToLayoutContainer</code></td>
+    <td>Update's <code>amp-list</code>'s layout to <code>layout="CONAINTER"</code>.</td>
+  </tr>
+  <tr>
+    <td><code>refresh</code></td>
+    <td>Refreshes data from the <code>src</code> and re-renders the list.</td>
+  </tr>
+</table>
+
+### amp-selector <a name="amp-selector-1"></a>
+
+<table>
+  <tr>
+    <th>Action</th>
+    <th>Description</th>
+  </tr>
+  <tr>
+    <td><code>clear</code></td>
+    <td>Clears all selections from a defined <code>amp-selector</code>.</td>
+  </tr>
+  <tr>
+    <td><code>selectUp(delta=INTEGER)</code></td>
+    <td>Moves the selection up by the value of `delta`. The default `delta` is set to 1.</td>
+  </tr>
+  <tr>
+    <td><code>selectDown(delta=INTEGER)</code></td>
+    <td>Moves the selection down by the value of `delta`. The default `delta` is set to -1.</td>
+  </tr>
+  <tr>
+    <td><code>toggle(index=INTEGER, value=BOOLEAN)</code></td>
+    <td>Sets the selected element's `selected` attribute if value is 'true', otherwise removes the attribute</td>
```

Comment:
Clarification: This action literally toggles the existence of the `selected` attribute on the target option specified by `index`. Concretely, when `value` is not present, it will always behave as follows: when `selected` attribute is present, remove; when `selected` attribute is absent, add. 

When the `value` param is specified, it's treated as "force select". So when `value === false`, always remove `selected` attribute. When `value === true`, always add `selected` attribute. If the attribute is present/absent already (i.e. `value === true` but the option already has the `selected` attribute) it's a no-op. 

I'm sure you have a better way of phrasing all this concisely for the sake of the documentation, but I thought I'd describe it specifically for the sake of completeness and accuracy.

Reply:
Thanks so much for this clarification. I've attempted to give it an update. LMKWYT!